test('Our first test', () => {
    throw new Error('F');
});
